src/components/components/HelpButton/HelpButton.component.jsx

```diff
@@ -0,0 +1,31 @@
+import React, { Component } from "react";
+import "./HelpButton.styles.styl";
+import classNames from "classnames";
+import {QuestionMark} from './icons.jsx';
+import Events from '../../../lib/Events.js';
+
+/**
+ * HelpButton component.
+ *
+ * @author Anna Botsula
+ * @category Components.
+ */
+class HelpButton extends Component {
+  render() {
+    const onClick = () => Events.emit('openhelpmodal');
+
+    return (
+          <button
+            id="helpButton"
+            className={classNames("helpButton")}
+            type="button"
+            onClick={onClick}
+            key="helpButton"
+          >
```

Comment:
the help button still needs:
* hover state like other buttons
* cursor of hand just like other buttons when hover

Reply:
created separate ticket: https://github.com/3DStreet/3dstreet-editor/issues/55